runtime/metering.go

```diff
@@ -0,0 +1,59 @@
+package runtime
+
+type MetringOperationType uint
+
+const (
+	// base [0-10)
+	OpTypeStatement MetringOperationType = iota
+	OpTypeLoop
+	OpTypeFunctionInvocation
+	_
+	_
+	_
+	_
+	_
+	_
+	_
+
+	// value operations [10-40)
+	OpTypeCompositeValueCreate
+	OpTypeCompositeValueCopy
```

Comment:
Transfer is copy/move. Probably not needed

Reply:
removed